src/lib.rs

```diff
@@ -51,3 +53,124 @@ pub use telio_lana;
 
 /// cbindgen:ignore
 pub use telio_firewall;
+
+pub use uniffi_libtelio::*;
+#[allow(clippy::panic, clippy::unwrap_used, clippy::expect_used, unwrap_check)]
+mod uniffi_libtelio {
```

Comment:
All of the conversion here should have tests.

Reply:
Added